src/OpenLoco/Windows/BuildVehicle.cpp

```diff
@@ -1331,11 +1328,11 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
 
         if (setRail)
         {
-            last_railroad_option = trackType | (isRoad ? (1 << 7) : 0);
+            LastGameOptionManager::setLastRailRoad(trackType | (isRoad ? (1 << 7) : 0));
```

Comment:
Perhaps add a note that railroad is also used for trams which are "roads" this is indicated by a (1<<7)